### ISSUE TITLE
Ironbane Page on Chrome Experiments needs update

### ISSUE THREAD

**Issue by eric-ycw** [ironbane/IronbaneServerLegacy#298]
http://www.chromeexperiments.com/detail/ironbane-mmo/

**Comment by ghost** [comment]
I second this. With 646 ratings, there's definitely some interest in IB. :3 Perhaps a few more videos, as well? Maybe that Rats trailer? :D

**Comment by eric-ycw** [comment]
Yep, more game trailers would be good :) Maybe we should aim for a goal, which we can make a official game trailer, and videos on updates.

**Comment by ghost** [comment]
In the mean time, everyone rate 5/5 stars every time you visit the page with a new IP. ;) I am! :D

Also, for game trailers, definitely need "screenshot mode."

If possible, some more options for such a mode so we can get some better FPS. 30 is nice but if I can record in 60, oooooh man that would be nice. Could make some x0.5 speed slo mo shots of ACTION.

I don't think an official official release trailer needs to be made until there's more content. A few more areas, maybe some more combat options, and a more powerful server.

Video updates aren't a bad idea, but then we need a dedicated editor and recorder.

**Comment by eric-ycw** [comment]
Is anyone working on this?

**Comment by ghost** [comment]
I have a new "From the author:" suggestion:
- Ironbane is an open source Action MMORPG played straight from your browser. It merges a 3D world and retro graphics style of pixel art and textures to create a unique look.
- Our plans include social networking integration so you can log in using your Twitter or Facebook account, game expansion through dungeons and world additions, a new, treeless perk system, and overall a more streamlined server.
- Log in as a guest to try out the game for yourself!

And for Tags:
- "3D Games Multiplayer WebGL 2D MMORPG MMO Action"

Then for screenshots we can grab some shiny new ones with AND without the GUI. :D

**Comment by TheRestIsATest** [comment]
"Our plans include social networking integration so you can log in using your Twitter or Facebook account"
What? I never heard of that

**Comment by ghost** [comment]
Oh.... Well it was mentioned somewhere. :P

**Comment by bsparks** [comment]
yep see #124

**Comment by TheRestIsATest** [comment]
Hm, is it a must have or are you just able to connect it?

**Comment by bsparks** [comment]
Optional
